changelogs/head.asciidoc

```diff
@@ -23,6 +23,7 @@ https://github.com/elastic/apm-server/compare/8.5\...main[View commits]
   When set to `true`, the batch will be processed asynchronously if the APM Server can
   service the incoming request immediately, otherwise a "503 queue is full" error will
   be returned {pull}8979[8979]
+- [EXPERIMENTAL] Add logs support to v2 intake API {pull}9068[9068]
```

Comment:
Better to follow the convention: `- experimental:[] Add logs support to v2 intake API {pull}9068[9068]`

It will let it appear as `preview` automatically (see [8.0 release notes](https://www.elastic.co/guide/en/apm/guide/current/release-notes-8.0.html))

<img width="1284" alt="Screenshot 2022-09-09 at 10 26 46" src="https://user-images.githubusercontent.com/5555349/189306842-4abd75dd-f6c1-42bd-aa62-6e368e44056e.png">

Reply:
Thanks, TIL. I have updated it.